Tests/SonarQube.Bootstrapper.Tests/ArgumentProcessorTests.cs

```diff
@@ -66,12 +108,20 @@ public void ArgProc_UrlIsRequired()
             // 2. Url is specified in the file -> ok
             logger = new TestLogger();
             IBootstrapperSettings settings = CheckProcessingSucceeds(logger, "/key:k1", "/name:n1", "/version:1.0", "/s:" + propertiesFilePath);
-            AssertExpectedUrl("http://filehost", settings);
+            AssertCommonSettings(
+                settings, 
+                "http://filehost", 
+                true, 
+                "\"/key:k1\" \"/name:n1\" \"/version:1.0\" \"/s:" + propertiesFilePath + "\"");
```

Comment:
It isn't really necessary to check all of the common settings all of the time, particularly if the other common arguments are just dummy settings that need to be provided to make the parsing succeed.

This test specifically covers some of the URL-related special cases. Parsing of the common arguments is tested elsewhere and isn't adding any value here; it just adds clutter and makes the real test cases harder to see.